pages/web-lit-basics-two.jsx

```diff
@@ -0,0 +1,143 @@
+var Illustration = require('../components/illustration.jsx');
+var ActivitySection = require('../components/activity-section.jsx');
+var Router = require('react-router');
+var Link = Router.Link;
+var OutboundLink = require('react-ga').OutboundLink;
+
+var CurriculumIntro = React.createClass({
+  render: function () {
+    return (
+      <div>
+        <h1>Web Literacy Basics II</h1>
+        <section className="intro">
+          <Illustration
+          height={204} width={204}
+          src1x="/img/pages/web-lit-basics-two/intro.jpg"
+          className="illustration-img-circle"
+          alt=""
+          caption={<a href="https://c2.staticflickr.com/6/5760/22431506387_43e85e71bd.jpg">cc-by-2.0 Mozilla Festival</a>}>
+            <h2>Dive deeper into the basics of composing webpages, designing for accessibility, sharing resources, using online media, and working open.</h2>
+          </Illustration>
+        </section>
+      </div>
+    );
+  }
+});
+
+var WebLitBasics = React.createClass({
+  render: function () {
+    return (
+      <section className="row web-lit-basics">
+        <div className="col-sm-12">
+          <h2>Learning Objectives</h2>
+          <p>
+            Learners will understand community practice, composing, designing for accessibility, open practices, sharing, and remixing and reflecting on how they and others create, use, and license media on the Web.
+          </p>
+        </div>
+      </section>
+    );
+  }
+});
+
+var WebLitBasicsTwo = React.createClass({
+  statics: {
+    pageTitle: 'Web Literacy Basics II',
+    pageClassName: 'web-lit-basics-two'
+  },
+  curriculum: [
+    {
+      title: "",
+      activities: [
+        {
+          title: "Why Do We Use the Web?",
+          image1x: "/img/pages/web-lit-basics-two/why-do-we-use-the-web.png",
+          originalImgSrc: "https://upload.wikimedia.org/wikipedia/commons/thumb/1/1b/Online_Survey_Icon_or_logo.svg/600px-Online_Survey_Icon_or_logo.svg.png",
+          subtitle: "Understanding Community Participation",
+          description: "Learners will create and research survey questions about their community's Web use, learning about <strong>collaborating, community participation, and open practices</strong>.",
+          link: "https://chadsansing.github.io/curriculum-testing/web-lit-basics-two/session01-why-do-we-use-the-web.html"
+        },
+        {
+          title: "The Web is a Tool for Learning",
+          image1x: "/img/pages/web-lit-basics-two/the-web-is-a-tool-for-learning.jpg",
+          originalImgSrc: "https://c1.staticflickr.com/1/88/234010602_0b42343b8d.jpg",
+          subtitle: "Understanding Navigating",
+          description: "Learners will use Web-native instructions to make a meme or build a maker project in real life, learning <strong>navigating, remixing, and community participation</strong>.",
+          link: "https://chadsansing.github.io/curriculum-testing/web-lit-basics-two/session02-the-web-is-for-making.html"
```

Comment:
there's a whole bunch of `http` links for chad's github pages, can we change all of those to `https`?